Dockerfile

```diff
@@ -1,5 +1,5 @@
 # Use an NVIDIA CUDA base image with Python 3
-FROM nvidia/cuda:11.6.2-base-ubuntu20.04
+FROM nvidia/cuda:11.6.2-cudnn8-runtime-ubuntu20.04
```

Comment:
The Dockerfile in the current repository will lead to the following issues:

```
Client 583ffd78-2bfe-4e1e-bbbc-8efed00e1904 connected
2024-05-20T08:38:53.837582891Z Could not load library libcudnn_ops_infer.so.8. Error: libcudnn_ops_infer.so.8: cannot open shared object file: No such file or directory
```

The current commit can fix this issue.